src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -140,39 +140,36 @@ describe('Menu', () => {
         <Menu
           id="test-menu"
           label="Test"
-          items={[{ label: 'Item 1', onClick }, { label: 'Item 2' }]}
+          items={[{ label: 'Item 1' }, { label: 'Item 2', onClick }]}
         />
       </Grommet>,
     );
     expect(container.firstChild).toMatchSnapshot();
 
-    // pressing down 3x: first opens the drop,
-    // second moves to the first suggestion
-    // third moves to the last suggestion
+    // pressing space: opens drop
+    // tab moves to menu label
+    // second moves to first item
+    // third moves to second item
     fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+      key: 'Space',
+      keyCode: 32,
+      which: 32,
     });
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
     });
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
     });
-
-    // moves to the first suggestion
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Up',
-      keyCode: 38,
-      which: 38,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
```

Comment:
Are we closing the Menu three times using Tab? 
Before these changes, it made sense to use the Tab in order to navigate within the items of the 'Open Menu', but when the Menu is closed how does that makes sense?

Reply:
We're not closing the menu, but the 'Close Menu' button refers to the Drop div that gets added to the DOM when a menu opens, and it seemed that the keyDown focus needed to refer to this element in order to advance through the menu items which are contained in this Drop. So, the menu is not closing but using that label text is helping to tell the test where the keyDown should occur. I am still strengthening my understanding of test writing, so another set of eyes may be helpful.